Makefile

```diff
@@ -0,0 +1,49 @@
+build:
+	docker run --rm -it \
```

Comment:
We did't know about the `docker.io/m0rf30/yap-ubuntu-focal` image, this is a great news! :tada: I'm going to use it immediately :ok_hand:

About the entrypoint script the issue seems to be a bit more complex, since involves changes in the `PKGBUILD` file we did't want to touch now.. could we postpone this improvements and keep the entrypoint script as first step to have a better developer experience on VMs deploy ?